augment-docker-image/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ###
 ### Dockerfile used for augmentation pipeline code
 ###
-FROM gcr.io/tensorflow/tensorflow:latest-gpu
+FROM tensorflow/tensorflow:1.15.2-gpu
 MAINTAINER Vincent Vanhoucke <vanhoucke@google.com>
-RUN pip install scikit-learn scikit-image opencv-python==3.2.0.8
+RUN pip install scipy==1.1.0 scikit-learn scikit-image opencv-python==3.2.0.8
```

Comment:
scipy 1.2.0+ removes scipy.imread and probably other functions needed by SEA. It was faster to install an older version than refactor the code for newer scipy